### ISSUE TITLE
Can we create event text cross the day component?

### ISSUE THREAD

**Issue by AlanChienHouzz** [kizitonwose/Calendar#464]
### Library information:
 - Version: `2.3.0`
 - View or Compose module: `Compose`

### Question:
Is it possible to make text cross the day like google calendar?
for instance: 

![Screenshot_20230429_184042](https://user-images.githubusercontent.com/107456743/235298513-7279e19f-49cb-4ef4-b09d-2a51f000cbb5.png)

**Comment by AlanChienHouzz** [comment]
It's possible to replace `monthBody` to achieve this I think. I'm trying to achieve that. Correct me if I'm wrong :)

**Comment by kizitonwose** [comment]
Could be tricky to do it that way. You may be better off with the [week row](https://github.com/kizitonwose/Calendar/blob/main/compose/src/main/java/com/kizitonwose/calendar/compose/CalendarMonths.kt#L60), but that would involve forking the project since it is not exposed.

**Comment by AlanChienHouzz** [comment]
@kizitonwose Thanks for replying. Yeah! It would be better if we can expose the whole week row. Do you think it's a worth effort to make this change?

**Comment by kizitonwose** [comment]
It's worth doing, this feature has been requested before but I just have not found the time.

**Comment by AlanChienHouzz** [comment]
Do you mind to let me try this enhancement? 
-
it's hard to find a best way to put the event cross the row item lol.
I was using Row with weight to put events. 

<img width="428" alt="image" src="https://user-images.githubusercontent.com/107456743/235833034-bd30a5b1-04cb-4507-9ef1-6f4f086c4079.png">

**Comment by kizitonwose** [comment]
Yes, feel free to implement it, let me know if you need any help along the way. Ideally, the week should be changed from just `Row` to  `Box -> Row -> External Overlay`. The outer box should only be added if the `External Overlay` composable is provided (not null).

**Comment by kizitonwose** [comment]
Hey @AlanChienHouzz Any updates? Need help?

**Comment by AlanChienHouzz** [comment]
@kizitonwose I need more time. Do you mind it..? 
Sorry. it's a little bit busy these days. long days..

**Comment by kizitonwose** [comment]
@AlanChienHouzz That's fine. I just wanted to know if you needed help, please take as much time as you need.

**Comment by DanteAndroid** [comment]
> Do you mind to let me try this enhancement?
> 
> -- it's hard to find a best way to put the event cross the row item lol. I was using Row with weight to put events.
> 
> <img alt="image" width="428" src="https://user-images.githubusercontent.com/107456743/235833034-bd30a5b1-04cb-4507-9ef1-6f4f086c4079.png">

How to achieve this, thanks

**Comment by arBespalov** [comment]
Hey @kizitonwose, thanks for the great library! I faced the same problem with long text crossing the days when tried to implement the multi-day events like Google Calendar does. Fortunately I was able to do that with the steps you guys described above (exposing week row to the outer world and then use weights to properly place the event). If you are still interesting in this feature, I can try to contribute then

![Image](https://github.com/user-attachments/assets/cad90056-676e-4e83-b44a-c881652960e6)
